### ISSUE TITLE
Fails caused by non-standard characters in file names

### ISSUE THREAD

**Issue by EveWright** [digital-preservation/csv-validator#339]
I have found that CSV Validator cannot locate some files listed in the CSV when using the checksum expression (MD5) if they have non-standard characters in the file names. I have found this in happens when the characters +, %, ~ or excessive white space appears in the file name. 

Is there a way around this so CSV Validator can locate the files (other than me removing the characters from the file names and updating my CSV file)?

**Comment by DavidUnderdown** [comment]
Again, do you have some specific examples of file names where this has happened, is your CSV coming from DROID again for these? (it's possible it could be an issue with the URLs being created there rather than with validator as such)

**Comment by EveWright** [comment]
Yes the CSV is coming from DROID for these as well. You can find some example files attached. Let me know if you require anything further.

[Character Issue.zip](https://github.com/digital-preservation/csv-validator/files/8315639/Character.Issue.zip)

**Comment by DavidUnderdown** [comment]
Thanks, I will see if I can recreate your issue. One question, are you testing against the FILE_PATH field from DROID or the URI field?

**Comment by EveWright** [comment]
Thanks David that's really helpful. I will give the URI a go
